influxdb_client/client/query_api.py

```diff
@@ -17,18 +17,21 @@
 from influxdb_client.client.flux_table import FluxTable, FluxRecord
 from influxdb_client.client.util.date_utils import get_date_helper
 from influxdb_client.client.util.helpers import get_org_query_param
+from typing import Callable
```

Comment:
Please add `Callable` to `from typing import List, Generator, Any, Union, Iterable`

![image](https://user-images.githubusercontent.com/455137/149348855-c32c0047-da96-47ee-b52a-c08bad746e7d.png)